cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -602,4 +603,35 @@ protected Properties getDefaultLogConfig() {
         }
         return properties;
     }
+
+    public ServiceAccount generateInitContainerServiceAccount() {
+        return new ServiceAccountBuilder()
+                .withNewMetadata()
+                    .withName(getInitContainerServiceAccountName(name))
+                    .addToLabels("app", "strimzi")
+                    .withNamespace(namespace)
+                .endMetadata()
+            .build();
+    }
+
+    public static String getInitContainerServiceAccountName(String name) {
+        return name + "-kafka-init";
```

Comment:
Actually, one more comment ... you are using the `name` variable here which already contains `clustername + SUFFIX` ... e.g. `my-cluster-kafka`. So after you add `-kafka-init` it actually creates a very ugly name `my-cluster-kafka-kafka-init`. Could we change this just to `name + "-init"`?

Reply:
All the other `xxxxName()` methods are using the `cluster` as parameter so that "-kafka-init" should be ok.
Another minor thing, we are using constants for all the SUFFIX.

Reply:
Just noticed that you are passing the cluster name at the `topicOperatorServiceAccountName`, so we could do the same here.